test/Core/calculators.jl

```diff
@@ -4,9 +4,13 @@ using NQCDynamics.Calculators
 using LinearAlgebra: tr, Diagonal, eigvecs, eigvals
 using RingPolymerArrays: RingPolymerArrays
 
-# For the allocation tests check against both backends as we can't be sure which is in use
+# For the allocation tests check against both backends as we can't be sure which is in use. 
 const MKL_EIGEN_ALLOCATIONS = 54912
```

Comment:
What does happen if somebody applies a Julia v1.7 version?

Reply:
I've changed the logic to do something for all Julia versions now